tensorflow/python/kernel_tests/math_ops/topk_op_test.py

```diff
@@ -55,7 +55,8 @@ def _validateTopK(self,
         # Do some special casing of equality of indices: if indices
         # are not the same, but values are floating type, ensure that
         # the values are within epsilon of each other.
-        if not np.issubdtype(np_expected_values.dtype, np.floating):
+        if not np.issubdtype(np_expected_values.dtype, np.floating) and \
```

Comment:
Sorry, can you remove the `\` and use brackets instead?  This is against our style guides.  See [here](https://google.github.io/styleguide/pyguide.html#32-line-length).  Thanks.

Reply:
Thanks @cantonios, fixed it.